src/lex.cpp

```diff
@@ -450,13 +450,36 @@ void lexer::parse_current_token() {
       if (this->input_[2] == '=') {
         this->last_token_.type = token_type::star_star_equal;
         this->input_ += 3;
+      } else if (this->input_[2] == '/') {
+        bool parsed_ok = this->test_for_regexp(1);
+        if (!parsed_ok) {
+          this->last_token_.type = token_type::star;
```

Comment:
Nit: I think the `!parsed_ok` branch deserves a comment:
```suggestion
          // We saw '**/'. Emit a '*' token now. Later, we will interpret the following '*/' as a comment. 
          this->last_token_.type = token_type::star;
```